samples/stream-reduce/chunk_sum/README.md

```diff
@@ -0,0 +1,7 @@
+**double_sequence**
+
+This example generates a sequence with the doubles of the first *n* natural numbers.
```

Comment:
I think that this description is not right. Looking at the code, this program generates the sequence of the firs "n" natural numbers, and then it computes their sums in chuncks of "windowsize" elements. Also, this pattern takes the offset as the number of non-overlapping items among windows. Finally, the sums of each window are printed.